General | Remove faulty ramlog storage update from cron job

MichaIng opened the pull request:
**Status**: Ready
- Whoopsie, excuse wrong version string

**Reference**: https://github.com/Fourdee/DietPi/issues/1947

**Commit list/description**:
+ General | Logging: Move dietpi-ramlog storage handling to "dietpi-logclear"

MichaIng committed:
v6.13

+ General | Logging: Move dietpi-ramlog storage handling to "dietpi-logclear"

MichaIng committed:
v6.12

+ DietPi-Logclear | Update "dietpi-ramlog" file structure, if enabled

Fourdee commented on dietpi/dietpi-logclear:
```diff
@@ -231,6 +231,19 @@
 
+		#Update dietpi-ramlog file structure, if enabled:
+ 		LOGGING_MODE=0
+ 		if [[ -r '/DietPi/dietpi/.installed' ]]; then
```
```-r```? 😕

MichaIng replied:
@Fourdee 
File is readable 😄: https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html

Fourdee commented on dietpi/dietpi-logclear:
```diff
@@ -231,6 +231,19 @@
+ 		fi
+		if (( $LOGGING_MODE == -1 || $LOGGING_MODE == -2 )); then
+
+			/DietPi/dietpi/dietpi-ramlog 1 &> /dev/null # Prevent confusing startup/shutdown output
```
I'd personally not bother with updating the disk. As we use this script during hourly clear, increases disk writes unnecessarily?

Ideally, let the reboot handle disk write, when its needed.

MichaIng replied:
@Fourdee 
Not sure. Thought it as kind of failsafe, in case of power loss or other reason for unintentional shutdown. If you have a server running for one month, installing the one or the other software, without reboot, and then just have an unlucky server crash, this will prevent you from loosing the log structure. Okay perhaps rare case and not too hard to resolve, as there are no actual logs lost...

However you prefer is fine for me, but if we do the log skeletons update regularly, then it fits best directly within `dietpi-logclear`, where the logs are changed.

Disk writes are very minimal, compared to what `dietpi-logclear` does anyway, as just empty files/folders are created. So nothing to worry in my opinion.

Fourdee replied:
@MichaIng 

Yep, my issue is, having it write to disk at the same time of clearing the logs, kind of defeats the purpose of RAMlog as a whole (to reduce SD writes).

I agree that users may loose log files if they manually install without reboot, however, ```dietpi-software``` use will reboot (unless automated install) anyway, which our general userbase will be using.

Hmmm. I know why you want it in, and I agree, but I believe in this case, adding SD writes during RAMlog clear, just doesn't fit for me.

---

So 1 file of <4K size uses a whole 4K IO block, 8*512.
```
root@DietPi:~# stat /var/log/wtmp
  File: /var/log/wtmp
  Size: 384             Blocks: 8          IO Block: 4096   regular file
Device: 1fh/31d Inode: 11807       Links: 1
Access: (0664/-rw-rw-r--)  Uid: (    0/    root)   Gid: (   43/    utmp)
Access: 2018-07-22 22:08:58.250000000 +0100
Modify: 2018-07-22 22:25:26.908338543 +0100
Change: 2018-07-22 22:25:26.908338543 +0100
 Birth: -
```
If I remember correctly, NAND needs to write to multiple 4K blocks to delete the file (MLC/TLC):
![image](https://user-images.githubusercontent.com/7077026/43050342-0cc1cdce-8dff-11e8-8962-865aaf639cc4.png)

http://www.flashstorage.com/flash-storage-technical-economic-primer/

MichaIng replied:
@Fourdee 
Okay, I think you are right, I send commit to remove it completely.

MichaIng committed:
v6.13

+ DietPi-Logclear | Revert log structure update due to: https://github.com/Fourdee/DietPi/pull/1949/files#r204255687

MichaIng commented:
I will merge this, as it is now trivial removal of faulty cron job extension 😄.
